Hazzle/Hazzle/Home/HomeViewController.swift

```diff
@@ -4,15 +4,39 @@
 //
 //  Created by 이지원 on 2020/10/23.
 //
-
 import UIKit
+import SnapKit
+import Then
 
 class HomeViewController: UIViewController {
-
+    let habits = ["수영하기 🏊‍♀️", "물 마시기 💦", "일기 쓰기 📓"]
+    let habitCnts = [13, 33, 53]
+    // MARK: - views
+    lazy var contentViewSize = CGSize(width: self.view.frame.width, height: self.view.frame.height+400)
```

Comment:
```suggestion
    lazy var contentViewSize = CGSize(width: self.view.frame.width, height: self.view.frame.height)
```
height 값에 400을 굳이 더해주지 않아도 될 거 같아요~ 정인님도 택현님이 올려주신 스크롤뷰 영상 참고하신 거 같은데, 그 영상에서 +400을 해준 건 스크롤뷰가 동작하는 지 확인하기 위해서 일부러 크기를 더 늘려준 거 같더라구요!

Reply:
앗 그러네요 수정했습니당

Reply:
Commit suggestion 누르면 리뷰어가 제안한 코드를 바로 커밋할 수도 있어요!

Reply:
오 완전신기하네요 바로커밋